utility/src/main/java/org/oppia/util/parser/HtmlParser.kt

```diff
@@ -45,12 +47,15 @@ class HtmlParser private constructor(
       htmlContent = htmlContent.replace("&amp;quot;", "")
     }
 
-    val imageGetter = urlImageParserFactory.create(htmlContentTextView, entityType, entityId, imageCenterAlign)
+    val imageGetter = urlImageParserFactory.create(
+      htmlContentTextView, entityType, entityId, imageCenterAlign)
 
-    val htmlSpannable = HtmlCompat.fromHtml(htmlContent, HtmlCompat.FROM_HTML_MODE_LEGACY, imageGetter, LiTagHandler()) as Spannable
+    val htmlSpannable = HtmlCompat.fromHtml(
+      htmlContent, HtmlCompat.FROM_HTML_MODE_LEGACY, imageGetter, LiTagHandler()) as Spannable
```

Comment:
We can have it like this
` val htmlSpannable = HtmlCompat.fromHtml(
      htmlContent, HtmlCompat.FROM_HTML_MODE_LEGACY, imageGetter, LiTagHandler()
    ) as Spannable
`